### PR TITLE
Explicitly document the root namespace behavior

### PR DESCRIPTION
**Pull request by m-czernek**
In this PR, I wanted to provide a small additional bit of documentation.

The concept of root namespace is quite important for the authz policies yet it is very easily missed, mentioned only once in the whole text.

This PR adds two small paras explaining why readers should care about authz policies. 

WDYT?

### CODE REVIEW

**Comment by m-czernek** [comment]
CI passing; thoughts anyone?

File: security/v1beta1/authorization.proto [hunk 1]
```diff
@@ -181,6 +181,15 @@ import "type/v1beta1/selector.proto";
 //    matchLabels:
 //      version: v1
 // ```
+//
+// When designing authorization policies, you can use the
+// `rootNamespace` option for default network behavior.
+// If you set `rootNamespace` in your `MeshConfig` resource, Istio defaults to
+// using authorization policies set in the root namespace when you create no
+// other policy in your namespace.
```
Comment (hzxuzhonghu) [inline security/v1beta1/authorization.proto:189]: Basically it it right, but I think you need to rework on the words

Comment (m-czernek) [reply security/v1beta1/authorization.proto:189]: Can you be more specific please? What exactly do you object to?

Comment (hzxuzhonghu) [reply security/v1beta1/authorization.proto:189]: I mean this is not clear for a newbie. 

Both the workload namespace and rootnamespace authoriz policy take effect, which is different from config like DestinationRule

**Pull request closed without merging**